# HG changeset patch
# User R. Andrew Ohana <andrew.ohana@gmail.com>
# Date 1332318123 -700
# Node ID 76bfd136f0f6d3ef53f66fec00caad393e225efd
# Parent  c9f8d2a1f12d2ebd12f182d857f947bd7e2da34c
Git layout patch for the root repository
add install script
add deps file
remove version checking -- unneeded now and will break build
deps actually unneeded
nevermind, we need this

diff --git a/spkg/install b/spkg/install
--- a/spkg/install
+++ b/spkg/install
@@ -59,326 +59,6 @@
     export SAGE_SPKG_OPTS="-f"
 fi
 
-###############################################################################
-# Setup environment variables pointing to the newest versions
-# of the base packages:
-###############################################################################
-newest="$PKGDIR/newest_version -base "
-
-SAGE_BZIP2=`$newest bzip2`
-export SAGE_BZIP2
-
-DIR=`$newest dir`
-export DIR
-
-PREREQ=`$newest prereq`
-export PREREQ
-
-###############################################################################
-# Setup environment variables pointing to the newest versions
-# of the standard packages:
-###############################################################################
-newest="$PKGDIR/newest_version"
-
-ATLAS=`$newest atlas`
-export ATLAS
-
-BLAS=`$newest blas`
-export BLAS
-
-BOEHM_GC=`$newest boehm_gc`
-export BOEHM_GC
-
-BOOST_CROPPED=`$newest boost-cropped`
-export BOOST_CROPPED
-
-CEPHES=`$newest cephes`
-export CEPHES
-
-CLIQUER=`$newest cliquer`
-export CLIQUER
-
-CDDLIB=`$newest cddlib`
-export CDDLIB
-
-PPL=`$newest ppl`
-export PPL
-
-ECL=`$newest ecl`
-export ECL
-
-CONWAY=`$newest conway_polynomials`
-if [ $? -ne 0 ]; then
-    echo "Error determining package name using spkg/standard/newest_version script."
-    exit 1
-fi
-export CONWAY
-
-ECLIB=`$newest eclib`
-export ECLIB
-
-ELLIPTIC_CURVES=`$newest elliptic_curves`
-export ELLIPTIC_CURVES
-
-CVXOPT=`$newest cvxopt`
-export CVXOPT
-
-DOCUTILS=`$newest docutils`
-export DOCUTILS
-
-ECM=`$newest ecm`
-export ECM
-
-EXTCODE=`$newest extcode`
-export EXTCODE
-
-F2C=`$newest f2c`
-export F2C
-
-GLPK=`$newest glpk`
-export GLPK
-
-ICONV=`$newest iconv`
-export ICONV
-
-#############
-# This is all for GNUtls crypto
-LIBGCRYPT=`$newest libgcrypt`
-export LIBGCRYPT
-
-OPENCDK=`$newest opencdk`
-export OPENCDK
-
-GNUTLS=`$newest gnutls`
-export GNUTLS
-
-LIBGPG_ERROR=`$newest libgpg_error`
-export LIBGPG_ERROR
-
-PYGMENTS=`$newest pygments`
-export PYGMENTS
-
-PYTHON_GNUTLS=`$newest python_gnutls`
-export PYTHON_GNUTLS
-############# end gnutls crypto
-
-PIL=`$newest pil`
-export PIL
-
-PATCH=`$newest patch`
-export PATCH
-
-LIBM4RI=`$newest libm4ri`
-export LIBM4RI
-
-LIBM4RIE=`$newest libm4rie`
-export LIBM4RIE
-
-FORTRAN=`$newest fortran`
-export FORTRAN
-
-FPLLL=`$newest libfplll`
-export FPLLL
-
-FREETYPE=`$newest freetype`
-export FREETYPE
-
-GAP=`$newest gap`
-export GAP
-
-G2RED=`$newest genus2reduction`
-export G2RED
-
-GD=`$newest gd`
-export GD
-
-GDMODULE=`$newest gdmodule`
-export GDMODULE
-
-GFAN=`$newest gfan`
-export GFAN
-
-GIVARO=`$newest givaro`
-export GIVARO
-
-MPIR=`$newest mpir`
-export MPIR
-
-GRAPHS=`$newest graphs`
-export GRAPHS
-
-GSL=`$newest gsl`
-export GSL
-
-IPYTHON=`$newest ipython`
-export IPYTHON
-
-LAPACK=`$newest lapack`
-export LAPACK
-
-LCALC=`$newest lcalc`
-export LCALC
-
-LIBPNG=`$newest libpng`
-export LIBPNG
-
-LINBOX=`$newest linbox`
-export LINBOX
-
-IML=`$newest iml`
-export IML
-
-JINJA2=`$newest jinja2` 
-export JINJA2 
-
-MATPLOTLIB=`$newest matplotlib`
-export MATPLOTLIB
-
-MAXIMA=`$newest maxima`
-export MAXIMA
-
-MERCURIAL=`$newest mercurial`
-export MERCURIAL
-
-MPFI=`$newest mpfi`
-export MPFI
-
-MOIN=`$newest moin`
-export MOIN
-
-MPFR=`$newest mpfr`
-export MPFR
-
-MPMATH=`$newest mpmath`
-export MPMATH
-
-NETWORKX=`$newest networkx`
-export NETWORKX
-
-NUMPY=`$newest numpy`
-export NUMPY
-
-NTL=`$newest ntl`
-export NTL
-
-#OPENSSL=`$newest openssl`
-#export OPENSSL
-
-#PYOPENSSL=`$newest pyopenssl`
-#export PYOPENSSL
-
-PALP=`$newest palp`
-export PALP
-
-PARI=`$newest pari`
-export PARI
-
-PEXPECT=`$newest pexpect`
-export PEXPECT
-
-POLYBORI=`$newest polybori`
-export POLYBORI
-
-PYNAC=`$newest pynac`
-export PYNAC
-
-CYTHON=`$newest cython`
-export CYTHON
-
-RATPOINTS=`$newest ratpoints`
-export RATPOINTS
-
-SAGENB=`$newest sagenb`
-export SAGENB
-
-SAGETEX=`$newest sagetex`
-export SAGETEX
-
-SPHINX=`$newest sphinx`
-export SPHINX
-
-SQLALCHEMY=`$newest sqlalchemy`
-export SQLALCHEMY
-
-SQLITE=`$newest sqlite`
-export SQLITE
-
-FLINTQS=`$newest flintqs`
-export FLINTQS
-
-FLINT=`$newest flint`
-export FLINT
-
-POLYTOPES_DB=`$newest polytopes_db`
-export POLYTOPES_DB
-
-PYCRYPTO=`$newest pycrypto`
-export PYCRYPTO
-
-PYTHON=`$newest python`
-export PYTHON
-
-R=`$newest r`
-export R
-
-# Needed when #9906 gets merged:
-# RPY=`$newest rpy2`
-# export RPY
-
-READLINE=`$newest readline`
-export READLINE
-
-RUBIKS=`$newest rubiks`
-export RUBIKS
-
-SAGE=`$newest sage`
-export SAGE
-
-SAGE_ROOT_REPO=`$newest sage_root` 
-export SAGE_ROOT_REPO 
-
-SAGE_SCRIPTS=`$newest sage_scripts`
-export SAGE_SCRIPTS
-
-SCIPY=`$newest scipy`
-export SCIPY
-
-TACHYON=`$newest tachyon`
-export TACHYON
-
-TWISTED=`$newest twisted`
-export TWISTED
-
-SCONS=`$newest scons`
-export SCONS
-
-SETUPTOOLS=`$newest setuptools`
-export SETUPTOOLS
-
-SINGULAR=`$newest singular`
-export SINGULAR
-
-SYMPOW=`$newest sympow`
-export SYMPOW
-
-SYMPY=`$newest sympy`
-export SYMPY
-
-SYMMETRICA=`$newest symmetrica`
-export SYMMETRICA
-
-TERMCAP=`$newest termcap`
-export TERMCAP
-
-ZLIB=`$newest zlib`
-export ZLIB
-
-ZODB=`$newest zodb3`
-export ZODB
-
-ZNPOLY=`$newest zn_poly`
-export ZNPOLY
-
 # If the user (or the Makefile) has set SAGE_PARALLEL_SPKG_BUILD=no,
 # then turn off parallel building by writing .NOTPARALLEL to
 # spkg/standard/parallel_make.cfg.  Otherwise clear that file.
